server/app/views/questiontypes/CheckboxQuestionFragment.html

```diff
@@ -0,0 +1,29 @@
+<div th:fragment="checkbox (question)" th:with="questionId=${#strings.randomAlphanumeric(8)}" th:id="${questionId}">
+    <fieldset class="usa-fieldset" th:aria-describedby="${questionId} + '-description'" th:with="multiSelectQuestion=${question.createMultiSelectQuestion()}">
+        <!-- Title and Help Text -->
+        <div th:replace="~{questiontypes/QuestionBaseFragment :: titleAndHelpText(${question}, ${questionId})}"></div>
```

Comment:
**thought:** I don't know if this makes sense based on it being within the div / fieldset, but we could also consider moving this into the ApplicantQuestionFragment file if it will be present in every question?

Reply:
Se we can't move the validation errors since that varies per question type. I think we could probably move the outer div and the title/help text, but I'd prefer to try that refactor in another PR (it will be a bit easier to visualize once we have a couple question types in)

Reply:
Sounds good, thanks!